recipe/meta.yaml

```
@@ -45,9 +45,9 @@ outputs:
         - libgomp      # [linux]
         - llvm-openmp  # [osx]
         - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
-      run:
-        - __cuda  # [cuda_compiler_version != "None"]
-        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
+      run: 
+        - __cuda  # [cuda_compiler_version != "None"] 
+        - {{ pin_compatible("cuda-version", min_pin="x") }}  # [(cuda_compiler_version or "None") != "None"] 
```

Noticed a minor style fix we can make here

```suggestion
      run:
        - __cuda  # [cuda_compiler_version != "None"]
        - {{ pin_compatible("cuda-version", min_pin="x") }}  # [(cuda_compiler_version or "None") != "None"]
```